### DISCUSSION TITLE
Is there a way to export "Additional Marker icons" with icons and colors?

### DISCUSSION THREAD

**Discussion by oyvindsolstad** [javalent/obsidian-leaflet#181] [General]
For some reason, I keep missing these. After a restart of the app, they are gone. And since I've spent some time picking icons and colors for them, it takes 10+ minutes setting it up again.

**Comment by valentine195** [comment]
They are stored in `<YOUR VAULT>/.obsidian/plugins/obsidian-leaflet-plugin/data.json`

They should _not_ be deleted, however, so something is going wrong. Let me investigate to see if anything could be happening.

**Comment by valentine195** [comment]
Could you turn on the developers console (Ctrl-Shift-I on Windows / Linux, Cmd-Alt-I on Mac) after a restart and tell me if there are any errors. It shouldn't happen (if it allowed you to create the marker icon) but it is possible that an invalid marker icon was somehow saved which then prevented the loading of the data.

Edit: Could you also send me your `data.json` file.

**Reply by oyvindsolstad** [reply]
I'll do it if it happens again! Where do I send you the data.json file?

**Reply by valentine195** [reply]
You can just paste it here if you're comfortable sharing that.

**Reply by oyvindsolstad** [reply]
Happened again. Lost all markers:

on 0.4.x Loaded
VM121:2 Loading Obsidian Leaflet v3.24.8
app.js:1 SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at t.<anonymous> (app.js:1)
    at app.js:1
    at Object.next (app.js:1)
    at s (app.js:1)
(anonymous) @ app.js:1
main.ts:39 loading obsidian-tracker plugin
main.ts:16 loading plugin "tasks"
main.ts:10 loading Hotkeys for starred files and searches
main.ts:31 Loading Note Refactor plugin
main.ts:82 loading FileExplorerNoteCount
main.ts:32 loading WordNet plugin
index.ts:356 Dataview: Parsed all file prefixes (0.015s)
index.ts:409 Dataview: Initialized CSV row cache (loading 0 CSV files).
index.ts:247 Dataview: Task & metadata parsing queued in 0.006s.

**Reply by valentine195** [reply]
Okay, thanks. Do you have a backup of the `data.json` file you can share with me? Something in it caused it to fail to parse as JSON, which is odd...

**Reply by oyvindsolstad** [reply]
Can I send it on e-mail? Some private info in it.

**Reply by valentine195** [reply]
You're in the OMG discord right (@ oyvind?) Just sent you a DM on there.
